src/settings/StaffSlips/components/EditSections/StaffSlipTemplateContentSection/StaffSlipTemplateContentSection.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  injectIntl,
+} from 'react-intl';
+import { Field } from 'react-final-form';
+
+import {
+  Col,
+  Row,
+} from '@folio/stripes/components';
+import { TemplateEditor } from '@folio/stripes-template-editor';
+import TokensList from '../../../TokensList';
```

Comment:
It looks like you separated stripes and local imports by empty line in other files, should this be also separated for consistency?

Reply:
@Terala-Priyanka could you consider the comment above?